File: gen_scripts/python3/scripts/e20.branch.py
#!/usr/bin/env python3
# prompt, get input, and convert to integer
number = int(input("Input a number: "))

# test number range
if number > 0:
  print("Number is greater than 0")
elif number < 0:
  print("Number is less than 0")
else:
  print("Number is 0")

File: lessons/gen_scripts/php/scripts/e20.branch.php
#!/usr/bin/env php
<?php
echo "Input a number: "; $Number = fgets(STDIN);

if ($Number > 0) {
  echo "Number is greater than 0\n";
} elseif ($Number < 0) {
  echo "Number is less than 0\n";
} else {
  echo "Number is 0\n";
}
?>
